DESCRIPTION

```diff
@@ -43,6 +43,7 @@ Authors@R: c(
 Depends: R (>= 3.1.0)
 Imports: methods
 Suggests: bit64, curl, R.utils, knitr, xts, nanotime, zoo, yaml
+SystemRequirements: zlib
```

Comment:
Does it mean that `zlib` has to be present to install data.table, even if one is not going to use compression? I hope not

Reply:
zlib necessary to compile. Seems like it's standard though. Windows and Mac users installing binaries from CRAN won't need to install anything special, afaik.
If there's a way to make zlib optional, would be ideal of course. But there's plenty of other higher priority things to do first. I'd wait to see if anyone reports difficulties due to missing zlib first.

Reply:
zlib necessary to compile. Seems like it's standard though. Windows and Mac users installing binaries from CRAN won't need to install anything special, afaik.
If there's a way to make zlib optional, would be ideal of course. But there's plenty of other higher priority things to do first. I'd wait to see if anyone reports difficulties due to missing zlib first.